ibis/backends/tests/test_generic.py

```diff
@@ -622,10 +622,10 @@ def test_table_info(alltypes):
     raises=PyDruidProgrammingError,
     reason="Druid only supports trivial unions",
 )
-@pytest.mark.xfail(
-    reason="For flink: IOException - Insufficient number of network buffers"
-)
 def test_table_info_large(con):
+    if con.name == "flink":
+        pytest.xfail(reason="IOException - Insufficient number of network buffers")
```

Comment:
Hi @cpcloud added the `xfail` like this, not sure if it is the best practice. Please let me know if you have other good implementations.

Reply:
No, this approach is not best practice for failing specific backends in Ibis. Take a look at the decorator on this test, and see if you can apply that approach here. There are many examples of this around the codebase.

Reply:
> No, this approach is not best practice for failing specific backends in Ibis. Take a look at the decorator on this test, and see if you can apply that approach here. There are many examples of this around the codebase.

Should I use the `notyet` marker?
```
@pytest.mark.notyet(
    ["flink"], reason="IOException - Insufficient number of network buffers"
)
```

Reply:
Yep, that looks good to me.

Reply:
I was looking for some `@pytest.mark.xfail()` , the `notyet` is a wrapper on the `xfail`.

Thanks for your feedback.